articles/quickstart/spa/angular2/02-user-profile.md

```diff
@@ -17,7 +17,7 @@ budicon: 292
 
 ## Request the Profile Scope
 
-The user's `access_token` requires a `scope` of `openid profile` to successfully retrieve their information. In the `WebAuth` instance, specify that you would like to ask for these scopes.
+To retrieve user information, request a scope of `openid profile` in the `WebAuth` instance. 
```

Comment:
@aaguiarz is WebAuth the same thing as auth0.WebAuth? If yes, I would suggest unifying the way it is mentioned. Also, is the instance of the webauth object always the same one or do you create multiple instances?